app/models/patient.rb

```diff
@@ -204,6 +204,14 @@ def recent_history_tracks
     versions.where(updated_at: 6.days.ago..)
   end
 
+  def all_versions(include_fulfillment)
+    all_versions = versions || []
+    all_versions += (external_pledges.includes(versions: [:item, :user]).map(&:versions).reduce(&:+) || [])
+    all_versions += (practical_supports.includes(versions: [:item, :user]).map(&:versions).reduce(&:+) || [])
+    # all_versions += fulfillment.includes(versions: [:item, :user]).versions || [] if include_fulfillment # next up
```

Comment:
I had a little trouble getting this to play nice, so I'd prefer to do it in isolation in a diff pr.

Reply:
cool cool, then this shouldnt be marked as fixing the issue, right?